apps/dashboard/app/assets/javascripts/active_jobs.js

```diff
@@ -1,25 +1,5 @@
-// This is a manifest file that'll be compiled into application.js, which will include all the files
-// listed below.
-//
-// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
-// or vendor/assets/javascripts of plugins, if any, can be referenced here using a relative path.
-//
-// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
-// compiled file.
-//
-// Read Sprockets README (https://github.com/sstephenson/sprockets#sprockets-directives) for details
-// about supported directives.
-//
-//= require jquery3
-//= require jquery-migrate-3.1.0.min.js
-//= require jquery_ujs
-//= require bootstrap-sprockets
-//= require dataTables/jquery.dataTables
-//= require dataTables/bootstrap/3/jquery.dataTables.bootstrap
-//= require lightbox2/javascripts/lightbox
-//= require oboe/oboe-browser.min
-//= require datatables_plugins/api/processing
-//= require_tree .
+//= require oboe/dist/oboe-browser.min
+//= require datatables.net-plugins/api/processing()
```

Comment:
just to double check that the parens is part of the filename `datatables.net-plugins/api/processing()` instead of `datatables.net-plugins/api/processing`

Reply:
```text
-rw-r--r-- 1 johrstrom PZS0714 1198 Mar 24 12:18 fnStandingRedraw.js
-rw-r--r-- 1 johrstrom PZS0714 1211 Mar 24 12:18 fnVisibleToColumnIndex.js
-rw-r--r-- 1 johrstrom PZS0714 1113 Mar 24 12:18 order.neutral().js
-rw-r--r-- 1 johrstrom PZS0714 1352 Mar 24 12:18 page.jumpToData().js
-rw-r--r-- 1 johrstrom PZS0714 1029 Mar 24 12:18 processing().js
-rw-r--r-- 1 johrstrom PZS0714  790 Mar 24 12:18 rows().generate().js
-rw-r--r-- 1 johrstrom PZS0714 1313 Mar 24 12:18 row().show().js
-rw-r--r-- 1 johrstrom PZS0714 1667 Mar 24 12:18 sum().js